src/django/api/models.py

```diff
@@ -1253,13 +1254,14 @@ def filter_by_query_params(self, params):
         facilities_qs = FacilityIndex.objects.all()
 
         if free_text_query is not None:
+            custom_text = format_custom_text(contributors[0], free_text_query)
             if switch_is_active('ppe'):
                 if embed is not None:
                     facilities_qs = facilities_qs \
                         .filter(Q(name__icontains=free_text_query) |
                                 Q(id=free_text_query) |
                                 Q(ppe__icontains=free_text_query) |
-                                Q(custom_text__icontains=free_text_query))
+                                Q(custom_text__contains=[custom_text]))
```

Comment:
Why are we using `contains` instead of `exact` or `iexact` here for an exact match?